scripts/set-default-contact/index.js

```diff
@@ -49,7 +49,7 @@ const getObjectFromMedicDb = async id => {
 const hasDefaultContact = async user => {
   if(typeof user.place === 'object' && typeof user.contact === 'object' &&  user.place._id  ){
     const place = await getObjectFromMedicDb(user.place._id);
-    return !(typeof place.contact === 'object' && !place.contact._id);
+    return !((typeof place.contact === 'object' && !place.contact._id) || typeof place.contact !== 'object');
```

Comment:
That being said, is there a special reason we are checking `place.contact === 'object'`?  Is there ever a valid case where `contact` would be a primitive value (like a UUID string)?  If so, should we consider a place with such a `contact` value to have a default contact or not?  

If we have no reason to expect that `place.contact` would ever be primitive (or if a primitive value for `place.contact` should never be considered a default contact), then I feel like this check can simply be:

```suggestion
    return place.contact && place.contact._id;
```

Reply:
With the version before this PR:

```javascript
!((typeof place.contact === 'object' && !place.contact._id)
```

I was trying to say, "If the place has a `contact` object, but it's `_id` is not set".  This is because I checked and  the CHT does this in the `default` config when it creates a new place:

```json
{
  _id: '80b7dc4a-434e-470d-b49f-a39a96b66c1b',
  _rev: '6-78f0646dc4df774c4498b874e0afd2e7',
  contact: { _id: '' },
}
```

However, then I found out that we can also NO `contact` object at all, say when you use `cht-conf` to import JSON docs:


```json
{
  _id: '80b7dc4a-434e-470d-b49f-a39a96b66c1b',
  _rev: '6-78f0646dc4df774c4498b874e0afd2e7'
}
```

So then I was like, "If the place has a `contact` object, but it's `_id` is not set OR `contact` isn't set at all' (aka not an object)" . 

And here we are!  

Let's check in real-time tomorrow (if you're free) to make sure we're both on the same page! :paperclip: